# HG changeset patch
# User Torsten <mttl@mailbox.org>
# Date 1522956311 -7200
#      Thu Apr 05 21:25:11 2018 +0200
# Branch stable
# Node ID 8cffe74b75416b6574ce4b3f30d1243f48e4d4cd
# Parent  6e935942e57a9812c61c2b4d59d72e7472571a55
fix missing entries in shortcut preferences for other languages

* shortcut-manager.cc (do_fill_treewidget): check the keyname instead the possibly
  translated key description

diff --git a/libgui/src/shortcut-manager.cc b/libgui/src/shortcut-manager.cc
--- a/libgui/src/shortcut-manager.cc
+++ b/libgui/src/shortcut-manager.cc
@@ -579,17 +579,17 @@ namespace octave
 
         QTreeWidgetItem *section = m_level_hash[sc.m_settings_key.section (':',0,0)];
 
         // handle sections which have changed and do not correspond to the
         // previously defined keyname
         if (section == editor_file)
           {
             // Closing tabs now in global tab handling section
-            if (sc.m_description.mid (0,6) == "Close ")
+            if (sc.m_settings_key.contains ("editor_file:close"))
               section = main_tabs;
           }
 
         QTreeWidgetItem *tree_item = new QTreeWidgetItem (section);
 
         // set a slightly transparent foreground for default columns
         QColor fg = QColor (tree_item->foreground (1).color ());
         fg.setAlpha (128);
